frontend/src/metabase/collections/components/BaseItemsTable.jsx

```diff
@@ -112,13 +113,13 @@ function BaseItemsTable({
     });
 
   return (
-    <table className="ContentTable" {...props}>
+    <Table {...props}>
       <colgroup>
-        <col span="1" style={{ width: "5%" }} />
-        <col span="1" style={{ width: "64%" }} />
+        <col span="1" style={{ width: "60px" }} />
```

Comment:
Remove `span` attributes if they are all set to `1`? 🤔

Reply:
Good call, fixed. Thank you!